build-wheelhouse/action.yml

```
@@ -89,7 +89,14 @@ runs:
     - name: "Verify library is properly installed and get its version number"
       shell: bash
       run: |
-        echo "library_version=$(python -c "from ${{ inputs.library-namespace }} import __version__; print(__version__)")" >> $GITHUB_ENV
+        library_version=$(python -c "from ${{ inputs.library-namespace }} import __version__; print(__version__)" || exit 1 )
+        if [ -z "$library_version" ]; then
+            echo "Problem getting the library version"
+            exit 1;
+        else
+            echo "The library version is: $library_version";
+        fi;
```

Well it seems to me that this "double exit" is oging to make parts of the code unusable... Shouldn't it be enough to just change line 92?
```suggestion
        library_version=$(python -c "from ${{ inputs.library-namespace }} import __version__; print(__version__)" || exit 1 )
```

---

Or the other way around, I don't really care. In fact, it's probable better
```suggestion
        library_version=$(python -c "from ${{ inputs.library-namespace }} import __version__; print(__version__)")
        if [ -z "$library_version" ]; then
            echo "Problem getting the library version"
            exit 1;
        else
            echo "The library version is: $library_version";
        fi;
```

---

That exit inside `library_version=$(... || exit 1)` works fine here, in my understanding it only exits out of the `$(command)` expression, all the lines below it should still be processed

---

In my preliminary tests here the exit inside `library_version=$(... || exit 1)` works fine, in my understanding it is only exiting out of the `$(command)` expression, not out of the bash shell.

(Had previously deleted this comment by mistake, sorry for the repeated emails/notifications)

---

Let's leave it as is. It's fine 😄

---

My thoughts were that the exit inside the `$()` should avoid having something in that variable as a consequence of the error... like the the traceback. But probably it does not make. I'm Ok with whatever you decide guys. You know better. Although it is already merged xD